store/sqlstore/utils.go

```diff
@@ -26,3 +30,11 @@ func MapStringsToQueryParams(list []string, paramPrefix string) (string, map[str
 
 	return fmt.Sprintf("(%v)", keys.String()), params
 }
+
+// finalizeTransaction ensures a transaction is closed after use, rolling back if not already committed.
+func finalizeTransaction(transaction *gorp.Transaction) {
+	// Rollback returns sql.ErrTxDone if the transaction was already closed.
```

Comment:
RE: "returns sql.ErrTxDone if the transaction was already closed" - nice, simplifies!